### PR TITLE
Start PMTCT postbirth messages at specific week

### PR DESCRIPTION
**Pull request by gsvr**

### CODE REVIEW

**Comment by gsvr** [comment]
@imsickofmaps Please review.

File: ndoh_hub/utils.py [hunk 1]
```diff
@@ -13,14 +15,22 @@ def get_pregnancy_week(today, edd):
     """ Calculate how far along the mother's prenancy is in weeks. """
     due_date = datetime.datetime.strptime(edd, "%Y-%m-%d")
     time_diff = due_date - today
-    time_diff_weeks = int(time_diff.days / 7)
+    time_diff_weeks = int(round(time_diff.days // 7))
```
Comment (imsickofmaps) [inline ndoh_hub/utils.py]: double `//`?

Comment (imsickofmaps) [inline ndoh_hub/utils.py]: got it... TIL

**Comment by imsickofmaps** [comment]
👍

**Pull request merged**